tests/src/ViewModel/ContentAsideTest.php

```diff
@@ -99,11 +99,13 @@ public function it_has_data()
     /**
      * @test
      */
-    public function it_must_have_a_title()
+    public function it_may_have_a_status()
     {
-        $this->expectException(InvalidArgumentException::class);
+        $contentAside = new ContentAside(new ContentAsideStatus("content aside"));
+        $this->assertSame('content aside', $contentAside['status']['title']);
 
-        new ContentAside(new ContentAsideStatus(''));
+        $contentAside = new ContentAside();
+        $this->assertNull($contentAside['status']);
```

Comment:
We have used the variables `$with` and `$without` fairly consistently throughout the tests in this library. 

See: https://github.com/elifesciences/patterns-php/blob/0b2f593568ec06abc4958cef10fdddad57f2ff68/tests/src/ViewModel/AboutProfileTest.php#L73-L82